assets/css/style.css

```diff
@@ -1344,3 +1344,20 @@ footer a:hover{
         width:170px !important;
     }
 }
+
+.dpgIndicators{
+    background-color:#ebf5fc;
+    display:inline-block;
+    padding:20px;
+    border-radius: 10px;
+    margin:5px 0;
+}
+
+.dpgIndicators i{
+    color:var(--text-color-dark);
+    margin:5px;
+}
+
+.sdgs img{
+    width:150px
```

Comment:
This hard-codes pixels into the design, so these changes will not be responsive based on screen size. This is similar feedback to what I gave in #146 and #147. Instead of adding more custom CSS changes, again I push for using the existing Bootstrap classes, which saves us the effort of defining and testing our own custom classes. Bootstrap has already done a lot of the hard work for us.